dlls/dinput/joystick_sdl.c

```diff
@@ -1295,12 +1295,14 @@ static BOOL _SetProperty(JoystickImpl *This, const GUID *prop, const DIPROPHEADE
         case (DWORD_PTR)DIPROP_FFGAIN:
         {
             LPCDIPROPDWORD pd = (LPCDIPROPDWORD)header;
-            int sdl_gain = MulDiv(This->sdldev->gain, 100, 10000);
+            int sdl_gain;
 
             TRACE("DIPROP_FFGAIN(%d)\n", pd->dwData);
 
             This->sdldev->gain = pd->dwData;
 
+            sdl_gain = MulDiv(This->sdldev->gain, 100, 10000);
```

Comment:
You could remove the `int sdl_gain;` above and initialize it here?

```
int sdl_gain = MulDiv(This->sdldev->gain, 100, 10000);
```

Reply:
Some C programming conventions recommend putting variable declarations at the start of blocks, that's why I've kept it there. I'm not sure what conventions Wine follows.